tests/acceptance/api_test.go

```diff
@@ -448,12 +451,14 @@ func TestApi(t *testing.T) {
 			cognitoUser2 := NewCognitoUser(t, tfOut, awsSession, accountID)
 			defer cognitoUser2.delete(t, tfOut, awsSession)
 			// Create an Account Entry
-			leasedAccountID := "123"
+			leasedAccountID := "123456789012"
 			timeNow := time.Now().Unix()
 			err := dbSvc.PutAccount(db.Account{
-				ID:             leasedAccountID,
-				AccountStatus:  db.Ready,
-				LastModifiedOn: timeNow,
+				ID:               leasedAccountID,
+				AccountStatus:    db.Ready,
+				AdminRoleArn:     fmt.Sprintf("arn:aws:iam::%s:role/adminRole", leasedAccountID),
+				PrincipalRoleArn: fmt.Sprintf("arn:aws:iam::%s:role/principalRole", leasedAccountID),
+				LastModifiedOn:   timeNow,
```

Comment:
Because we are loading records directly into the system and not using the APIs I did a little more work to clean up the records to match expectations. This is a short term fix to a larger problem.